### DISCUSSION TITLE
I don't know how to use the CLI tool.

### DISCUSSION THREAD

**Discussion by NapoleonPR0** [anidl/multi-downloader-nx#93] [Q&A] [answered]
Hello how are all my friends in these comments, I would like to communicate with the developer of this Funimation videos script, I would like to tell him that a group could be created in mesemger, in whatsapp or in tekegram or in Discord, I have doubts about his commands I have tried everything possible and I do not get the videos to download from the Funimation platform, I was looking for two days and I gave up, I am too sad, I did not want my time to be wasted.

**Comment by DragonVsKira** [comment]
The video that I posted was only how to get the video ID number to download the anime I did not show how to download it only the ID part if that help ps I do not use any social media

**Comment by OryWareKyrie** [comment, marked as answer]
I think it's kind of unlikely for the izuco to open a channel on discord but I can try to help you. call me in discord ```#4772```

**Reply by izu-co** [reply]
Currently working on it tho :(

**Reply by OryWareKyrie** [reply]
> Currently working on it tho :(

Good!!!
